src/pages/api/account/register.ts

```diff
@@ -1,6 +1,40 @@
+import { hash } from 'bcrypt';
 import { NextApiRequest, NextApiResponse } from 'next';
+import { stringify } from 'querystring';
+import container from '../../../lib/container'
+import * as st from '../../../lib/domain/student'
 
-export default async function createAccount(req: NextApiRequest, res: NextApiResponse) {
+export default async function register(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+//   const db = await sqlite.open('./mydb.sqlite');
 
-    res.json({hello: 'world', id: req.query.id, method: req.method});
-} 
\ No newline at end of file
+  const ser = container.resolve(st.StudentService)
+
+  if (req.method === 'POST') {
+    hash(req.body.password, 12, async function(err, hash) {
+      // Store hash in your password DB.
+
+      const output = await ser.registerStudent(
+        {
+            name: req.body.name,
+            email: req.body.email,
+            universityId: req.body.universityId,
+            studyProgramId: req.body.studyProgramId,
+            passwordHash: hash
+        }
+      )
+    //   const statement = await db.prepare(
+    //     'INSERT INTO Student (name, email, password) values (?, ?, ?)'
```

Comment:
I think just by removing comments you are all set

Reply:
Yeah, just left that in for reference